services/motion/builtin/moveRequest.go

```diff
@@ -176,6 +177,10 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 						if stopErr := kb.Stop(ctx, nil); stopErr != nil {
 							return moveResponse{err: errors.Wrap(err, stopErr.Error())}
 						}
+						// If the error was simply a cancellation of context return without erroring out
```

Comment:
[q]: shouldn't this be inside the err != nil conditional? this way we know which type of moveresponse to return